ClientApp/src/components/LoginRegisterComponents/Signup.js

```diff
@@ -95,7 +95,7 @@ export default class SignUp extends React.Component {
       }
 
 addUser(newUser) {
-    fetch('https://localhost:44312/api/UserInformations', {
+    fetch('https://localhost:5001/api/UserInformations', {
```

Comment:
I would rather split  port switching to different PR, as it would be easier for reviewers to take a look at it